src/api.js

```diff
@@ -608,6 +608,13 @@ api.declare({
 }, async function(req, res) {
   var taskId  = req.params.taskId;
   var taskDef = req.body;
+  var clientId = await req.clientId();
+
+  if (taskDef.hasOwnProperty(clientId) && taskDef.clientId !== clientId) {
```

Comment:
This should have a comment, and move it into `patchAndValidateTaskDef`

Reply:
@jonasfj Ok! :D modified